.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Resolves #
+
+
+
+#### Pre-Merge Checklist:
+- [ ] Code is documented in doxygen format
+- [ ] Code has automated tests
+- [ ] Zero compiler warnings
+- [ ] Formatted with `clang-format`
```

Comment:
Same with compiler warnings, but not everyone knows that.
Good point, on the other hand, If I take it out, I may get messages saying "why is travis failing?"